src/common/components/cards/instance-details-footer.tsx

```diff
@@ -43,12 +42,13 @@ export const InstanceDetailsFooter = NamedFC<InstanceDetailsFooterProps>('Instan
     const issueDetailsData: CreateIssueDetailsTextData = deps.unifiedResultToIssueFilingDataConverter.convert(result, rule, targetAppInfo);
 
     const renderKebabMenu = () => {
+        const kebabMenuArialLabel: string = `More Actions for card ${result.identifiers.identifier}in rule ${rule.id}`;
```

Comment:
Missing space after placeholder in template string?
```suggestion
        const kebabMenuArialLabel: string = `More Actions for card ${result.identifiers.identifier} in rule ${rule.id}`;
```
(Snapshots will need to be updated if suggestion is accepted)

Reply:
Updated